api-client/src/endpoints/__tests__/createTrackingEndpoint.spec.ts

```diff
@@ -24,25 +24,22 @@ describe('createTrackingEndpoint', () => {
     timestamp: '2020-01-20T00:00:00.000Z'
   }
   it('should throw fetch error if fetch fails', async () => {
-    ;(fetch as unknown as FetchMock).mockRejectOnce(() =>
-      Promise.reject(new Error('Das Internet ist kaputt!!!1!!!11elf!'))
+    const error = new Error('Das Internet ist kaputt!!!1!!!11elf!')
+    ;(fetch as unknown as FetchMock).mockRejectOnce(() => Promise.reject(error))
```

Comment:
just change the import to `import fetch from 'jest-fetch-mock'` them you don't need to parse this object